src/Ormolu/Printer/Meat/Type.hs

```diff
@@ -145,7 +146,7 @@ p_hsType' multilineArgs = \case
   where
     isPromoted = \case
       HsTyVar _ IsPromoted _ -> True
-      HsExplicitListTy _ _ _ -> True
+      HsExplicitListTy {} -> True
       HsExplicitTupleTy _ _ -> True
```

Comment:
`HsExplicitTupleTy` should also be changed then.

Reply:
Done